core/unit_test/TestReducers_d.hpp

```diff
@@ -79,6 +79,51 @@ TEST(TEST_CATEGORY, reducers_half_t) {
   TestReducers<ThisTestType, TEST_EXECSPACE>::test_prod(25);
 }
 
+// TODO: File a bug report for this?
```

Comment:
@crtrott, @dalg24: Should a bug report for the OPENMP backend be filed for this? From what I observed, it looks like there's a data race in reductions with scalars < 4 bytes in size.

Reply:
Yes please

Reply:
Filed #4719